.github/CONTRIBUTING.md

```diff
@@ -32,15 +32,48 @@ traefik*
 
 - You need `go` v1.8+
 - It is recommended you clone Træfik into a directory like `~/go/src/github.com/containous/traefik` (This is the official golang workspace hierarchy, and will allow dependencies to resolve properly)
-- This will allow your `GOPATH` and `PATH` variable to be set to `~/go` via:
+- Set your `GOPATH` and `PATH` variable to be set to `~/go` via:
```

Comment:
(I thought I had made the following comment already, but I can't see it.)

Given that we require Go 1.8 which doesn't need a set `GOPATH` variable anymore, is this part still relevant?